ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -1513,10 +1514,11 @@ def verify_provider_storagecluster(sc_data):
         else:
             assert item["operator"] == "DoesNotExist"
     log.info(f"storageProviderEndpoint: {sc_data['status']['storageProviderEndpoint']}")
-    assert re.match(
-        "(\\d+(\\.\\d+){3}|[\\w-]+(\\.[\\w-]+)+):\\d{5}",
-        sc_data["status"]["storageProviderEndpoint"],
-    )
+    if version.get_semantic_ocs_version_from_config() >= version.VERSION_4_11:
+        reg = "(\\w+\\-\\w+\\.\\w+\\-\\w+\\-\\w+\\.elb.amazonaws.com):50051"
+    else:
+        reg = "(\\d+(\\.\\d+){3}|[\\w-]+(\\.[\\w-]+)+):\\d{5}"
```

Comment:
Why do regex patterns differ for provider and consumer? In this one is not checked a port.

Reply:
Yes, I saw that it was different patterns with ODF 4.10. But I think you are right, and it should be the same.

Reply:
Fixed